content/Arduino Cloud/Cloud Editor/Open-the-Cloud-Editor.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Open the Cloud Editor"
+---
+
+## Open a sketch
+
+To open a sketch in the Cloud Editor, follow these steps:
+
+1. Go to [app.arduino.cc/sketches](app.arduino.cc/sketches).
+
+1. Click *Create* > *New Sketch* in the top-right.
```

Comment:
It occurred to me that we can also include the option to open a saved sketch here:

```suggestion
1. Choose a sketch to open in the Cloud Editor:

    * **Create a new sketch:** Click on the Create button in the top-right corner and choose New Sketch.
    * **Open a saved sketch:** Click on a sketch you want to open.
```